exporter/elasticsearchexporter/factory.go

```diff
@@ -129,6 +143,11 @@ func createMetricsExporter(
 	cf := cfg.(*Config)
 	logConfigDeprecationWarnings(cf, set.Logger)
 
+	// Workaround to avoid rejections from Elasticsearch
+	// TSDB does not accept 2 documents with the same timestamp and dimensions
+	cf.BatcherConfig.MaxSizeConfig.MaxSizeItems = 0
```

Comment:
Should we update a copy of BatcherConfig here? @lahsivjar had some thoughts about making the exporter a SharedComponent so the min/max limits are shared across signals.

Reply:
BatcherConfig is a value, not a pointer, and it is copied by definition. If we want to make a copy, we would need to make a copy around the line: `cf := cfg.(*Config)`

Reply:
`cf` is a pointer, so updating `cf.BatcherConfig` will change a value that is shared by other holders of `cf`. So yeah what I mean is basically:

```go
batcherConfig := cf.BatcherConfig // make a copy for mutation
batcherConfig.MaxSizeConfig.MaxSizeItems = 0 // ...

...

    exporterhelper.WithBatcher(batcherConfig),
```

Reply:
Nice one. Could be a problem if other parts of the exporter other than Batcher uses the batcherConfig since the original cfg is passed and stored in the exporter, but it should be fine for now.